www/index.html

```diff
@@ -310,7 +310,7 @@ <h4 id="FAT">Downloading and storing large archives</h4>
                     </p>
                     <p style="text-align: right"><a href="#contents">↑ Back to Contents</a></p>
                     
-                    <h4 id="modes">ServiceWorker and JQuery Modes</h4>
+                    <h4 id="modes">Service Worker and JQuery Modes</h4>
```

Comment:
This one should be `ServiceWorker` because the phrase is short for `ServiceWorker Mode and JQuery Mode`.